doc/REFERENCE.md

```diff
@@ -387,16 +402,50 @@ i.e.
 Normally `_OpenSpaceNet_` will output its status to the console even if a log file is specified. If
 this is not desired, console output can be suppressed by specifying this option.
 
+<a name="details" />
+
+## Further Details
+
+
+<a name="input" />
+
+### Image Input
+
+_OpenSpaceNet_ is able use either a geo-registered local image or one of number of web maping servcies as input. Depending on the 
+input source, different command line arguments apply. To select which source to use, one of two options must be present:
+
+* `--service <service name>` To load imagery from a web service.
+* `--image <path>` To load a local image file.
+
+Depending on which source you use, different arguments apply.
+
+| source                | token    | credentials | map-id   | zoom     | bbox     | max-connections | url      | use-tiles |
+|-----------------------|----------|-------------|----------|----------|----------|-----------------|----------|-----------|
+| `--service dgcs`      | Required | Required    |          | Optional | Required | Optional        |          |           |
+| `--service evwhs`     | Required | Required    |          | Optional | Required | Optional        |          |           |
+| `--service maps-api`  | Required |             | Optional | Optional | Required | Optional        |          |           |
+| `--service tile-json` |          | Optional    |          | Optional | Required | Optional        | Required | Optional  |
+| `--image <path>`      |          |             |          |          | Optional |                 |          |           |
+
+<a name="size" />
+
+## Size Parameters
+
+Some models may require that the imagery be resized or padded to fit the target 
+model.  When performing these actions, use `--resampled-size` and `--window-size` 
+to add padding and resizing, respectively.  See the image below for guidance.
+
+![Resizing parameters](./modelsize.svg)
```

Comment:
Apparently, on my machine this now renders with black in places of transparent background?

Reply:
Yeah, I couldn't figure that one out. I'll play around some more.